grc/core/Config.py

```diff
@@ -8,6 +8,7 @@
 import os
 from os.path import expanduser, normpath, expandvars, exists
 from collections import OrderedDict
+from gnuradio import gr
```

Comment:
Generally, grc/core/ tries to avoid importing GNU Radio outside a try/catch block. (I think this is to ease testing and to keep GRC separable.)

anyways, you don't seem to use gr in this file?